src/redux/books/books.js

```diff
@@ -0,0 +1,35 @@
+// Action Types
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
```

Comment:
- [ ] You have done nice work so far. To improve the project, kindly make the actions you are creating have strings that reflect the duck pattern. For Example, `const ADD_BOOK = bookstore/books/ADD_BOOK`. 👍